tests/unit/api/c/source/test_source.cpp

```diff
@@ -77,8 +112,25 @@ static SacnMcastInterface kTestNetints[NUM_TEST_NETINTS] = {{{kEtcPalIpTypeV4, 1
                                                             {{kEtcPalIpTypeV4, 3u}, kEtcPalErrOk}};
 static const size_t kTestReturnSize = 1234u;
 static const int kTestReturnInt = 5678;
-static const SacnSourceUniverseNetintList kTestNetintList = {kTestHandle, kTestUniverse, kTestNetints,
-                                                             NUM_TEST_NETINTS};
+static const SacnSourceUniverseNetintList kTestNetintLists[NUM_TEST_NETINT_LISTS] = {
+    {kTestHandle, kTestUniverse, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle, kTestUniverse2, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle2, kTestUniverse, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle2, kTestUniverse2, kTestNetints, NUM_TEST_NETINTS}};
+static const SacnSourceUniverseNetintList kTestInvalidNetintLists1[NUM_TEST_INVALID_NETINT_LISTS_1] = {
+    {kTestHandle, kTestUniverse, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle, kTestUniverse2, kTestNetints, NUM_TEST_NETINTS}};
+static const SacnSourceUniverseNetintList kTestInvalidNetintLists2[NUM_TEST_INVALID_NETINT_LISTS_2] = {
+    {kTestHandle, kTestUniverse, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle2, kTestUniverse2, kTestNetints, NUM_TEST_NETINTS}};
+static const SacnSourceUniverseNetintList kTestInvalidNetintLists3[NUM_TEST_INVALID_NETINT_LISTS_3] = {
+    {kTestHandle, kTestUniverse, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle, kTestUniverse2, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle2, kTestUniverse, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle2, kTestUniverse2, kTestNetints, NUM_TEST_NETINTS},
+    {kTestHandle2, kTestUniverse3, kTestNetints, NUM_TEST_NETINTS}};
```

Comment:
Is there a better way to organize these test values than 100 lines of constant definitions? The answer might well be no... but this is pretty hard to read and parse so it might be worth thinking about.

Reply:
https://github.com/ETCLabs/sACN/pull/47/commits/2a7990d5464cdaf6a2e12a2595c12f309dcf3c67